app/Http/Controllers/ApiTweetController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+use App\Models\Tweet;
+use App\Models\User;
+use App\Http\Controllers\ApiUserController;
+use App\Http\Requests\UserRequest;
+
+class ApiTweetController extends Controller
+{
+    //認証もバリデーションも必要ないため、普通のRequesクラス
+    public function getAllTweets(Request $request){
+        $tweets = Tweet::get()->toJson(JSON_PRETTY_PRINT);
+        return response($tweets, 200);
+    }
+
+    public function createTweet(UserRequest $request){
+        //ツイートの処理
+        $input = $request->validated();
+        $input_content = $input["content"];
```

Comment:
[Should]
多分ここはこれで取れる気がする。試してみて欲しいです。

```suggestion
        $input_content = $request->input('content');
```

RequestクラスとしてControllerに受け渡された時点で、Requestクラスに記載されたバリデーションは全て実行されているはずです。